src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -60,9 +62,15 @@
     <p>Click the <a class="btn btn-xs navbtn-inverse" ng-click="kbnTopNav.open('add'); toggleAddVisualization = !toggleAddVisualization" aria-label="Add visualization">Add</a> button in the menu bar above to add a visualization to the dashboard. <br/>If you haven't setup a visualization yet visit <a href="#/visualize" title="Visualize">"Visualize"</a> to create your first visualization.</p>
   </div>
 
+  <div ng-show="showViewHelpText()" class="text-center start-screen">
```

Comment:
I don't really like either - `can` sounds off to me because it's not can or can not, but should or should not.  Also don't like `is`, because it makes it sound like it's actually checking if the help text is visible, but it's not initially.

What about `getShowViewHelpText` to distinguish a question (`showViewHelpText?`) from a command (`showViewHelpText!`)? Alternatively, `getShouldShowViewHelpText`, but that's quite verbose.

Thoughts?